### PR TITLE
Handle incorrect content type

### PR DESCRIPTION
**Pull request by kirgene**

### CODE REVIEW

**Review by ksatya77 — APPROVED** [review]
looks good

File: src/lib/request.js [hunk 1]
```diff
@@ -62,6 +62,11 @@ const request = async (opts) => {
                 if (responseType === ResponseType.Text || !result.length) {
                     result = result.toString();
                 } else if (responseType === ResponseType.JSON) {
+                    const contentType = res.headers['content-type'];
+                    if (!/^application\/json/.test(contentType)) {
```
Comment (msk-) [inline src/lib/request.js:66]: Why not this?
```suggestion
                    if (!/^application\/json$/.test(contentType)) {
```

**Review by eoln — APPROVED** [review]
lgtm. take a look on @msk- suggestion

File: src/lib/request.js (hunk 1 above)
Comment (kirgene) [reply src/lib/request.js:66]: @msk- could be also `Content-type: application/json; charset=utf-8`

**Pull request merged**